Sources/SwiftGenCLI/templates/xcassets/swift5.stencil

```diff
@@ -129,7 +129,7 @@
   {% endfor %}
 {% endmacro %}
 // swiftlint:disable identifier_name line_length nesting type_body_length type_name
-{{accessModifier}} enum {{enumName}} {
+{{accessModifier}} enum {{enumName}}: Sendable {
```

Comment:
I assume the template should be compatible with all Swift5 variants. Therefore I suggest the following:

```suggestion
#if swift(>=5.7)
{{accessModifier}} enum {{enumName}}: Sendable {
#else
{{accessModifier}} enum {{enumName}} {
#endif
```

If I'm wrong - just ignore the comment